tests/test_planner_unit_orchestration.py

```
@@ -0,0 +1,453 @@
+                    slug="start-here",
+                    section="Start Here",
+                    description="d",
+                    owned_files=list(passed_scan.file_summaries),
+                    generation_hints={"is_introduction_page": True},
+                )
+            ],
+            nav_structure={"Start Here": ["start-here"]},
+            skipped_files=[],
+        )
+
+    with patch("deepdoc.planner.engine.run_phase2_scans", side_effect=lambda s, c, l, repo_root: s), \
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)